### PR TITLE
[eclipse/xtext-core#829] Update to Gradle Wrapper 4.9

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext-core#829] Update to Gradle Wrapper 4.9
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
Dont merge yet CQ is still outstanding

File: org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.full/org.xtext.example.full.parent/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
+#Thu Aug 23 17:08:13 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
```
Comment (szarnekow) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.full/org.xtext.example.full.parent/gradle/wrapper/gradle-wrapper.properties]: Should we use gradle-4.9-all.zip instead to sport debugging of the build?

**Comment by cdietrich** [comment]
am not sure. i just a `gradle wrapper --gradle-version=4.9` and i am not sure what needs to be called for that?!?

**Comment by szarnekow** [comment]
We would need to set a different distribution type in the wrapper task. Not sure it's worth it though:
https://docs.gradle.org/current/dsl/org.gradle.api.tasks.wrapper.Wrapper.html#org.gradle.api.tasks.wrapper.Wrapper:distributionType

**Pull request merged**